h/accounts/test/views_test.py

```diff
@@ -349,7 +349,7 @@ def test_forgot_password_creates_no_activations_when_validation_fails(activation
 @patch('h.accounts.views.reset_password_link')
 @forgot_password_fixtures
 def test_forgot_password_generates_reset_link(reset_link, authn_policy):
-    request = DummyRequest(method='POST')
```

Comment:
Using a dummy request object is almost always a better idea than using a dumb Mock, because it will raise if we try and access attributes on it that aren't supposed to exist.

You appear to have done this so you can set `request.authenticated_userid` to `None`, but that's already done by the `authn_policy` fixture.